airflow/jobs/local_task_job.py

```diff
@@ -79,11 +79,7 @@ def signal_handler(signum, frame):
             """Setting kill signal handler"""
             self.log.error("Received SIGTERM. Terminating subprocesses")
             self.on_kill()
-            self.task_instance.refresh_from_db()
-            if self.task_instance.state not in State.finished:
-                self.task_instance.set_state(State.FAILED)
-            self.task_instance._run_finished_callback(error="task received sigterm")
-            raise AirflowException("LocalTaskJob received SIGTERM signal")
+            self.handle_task_exit(128 + signum)
```

Comment:
what's 128 + signum?

Reply:
Instead of writing 143, which is sigterm exit code, I divided it so I can use signum to show where the code comes from. Also see this
https://tldp.org/LDP/abs/html/exitcodes.html

I can write it 143 if it’s better